client/src/app/(modules)/practices/(main)/(details)/parsers.ts

```diff
@@ -4,6 +4,11 @@ import { Practice } from '@/types/generated/strapi.schemas';
 
 import type { FieldType } from './field';
 
+// Strapi doesn't allow the type to be an array, so we are temporarily asserting it here
```

Comment:
Can't you use the type defined in `src/app/(modules)/practices/(main)/types.ts` instead of duplicating it?